Podfile

```diff
@@ -28,7 +28,7 @@ target 'TCAT' do
     
     # UI Frameworks
     pod 'DZNEmptyDataSet'
-    pod 'NotificationBannerSwift', '~> 1.7'
+    pod 'NotificationBannerSwift', :git=> 'https://github.com/mattbarker016/NotificationBanner.git'
```

Comment:
what are your thoughts on moving all of these types of forks to be on appdev repo instead of personal? When you leave what will happen? I think you can change ownership. This would be for all repos across all of appdev.